components/skus/browser/rs/wasm/src/lib.rs

```diff
@@ -137,35 +133,40 @@ impl JSSDK {
         let sdk = self.sdk.clone();
 
         let future = async move {
-            let window: ChromeWindow = web_sys::window()
-                .ok_or("couldn't get window")?
-                .dyn_into()
-                .unwrap();
+            let window: ChromeWindow =
+                web_sys::window().ok_or("couldn't get window")?.dyn_into().unwrap();
             if let Some(chrome) = window.chrome() {
                 if let Some(skus) = chrome.braveSkus() {
                     return JsFuture::from(skus.refresh_order(&order_id)).await;
                 }
             }
 
-            let order = sdk
-                .refresh_order(&order_id)
-                .await
-                .map_err(|e| e.to_string())?;
+            let order = sdk.refresh_order(&order_id).await.map_err(|e| e.to_string())?;
             Ok(JsValue::from_serde(&order).unwrap())
         };
 
         future_to_promise(future)
     }
 
+    #[wasm_bindgen]
+    pub fn submit_receipt(&self, order_id: String, receipt: String) -> js_sys::Promise {
+        let sdk = self.sdk.clone();
+
+        let future = async move {
+            let resp = sdk.submit_receipt(&order_id, &receipt).await.map_err(|e| e.to_string())?;
+            Ok(JsValue::from_serde(&resp).unwrap())
```

Comment:
better to avoid unwrap if possible

Reply:
updated in https://github.com/brave/brave-core/pull/13820/commits/26007fcae9d1b2f2c1519a54a13a8fdf140a278b to no longer have unwrap as suggested.